CS3219-AY2223S1/cs3219-project-ay2223s1-g12
# Implement storing and displaying of pictures for questions

Closes #130 

**Summary**
* Questions currently only consist of string data, which may not be sufficient as some questions require image examples to help the user better understand the question.
* Update Question Bank and Question Schema to support base64 encoded image data.
* Implement rendering of image data if it exists in `RoomPage.js` and `QuestionDisplay.js`.
* To test with a question that has an image: 
   * In `question-controller.js`, update line 29 to `await _findQuestionRandomly(level, 'Rotting Oranges');` to force load that question. 
   * When selecting question difficulty in the frontend, select Medium Difficulty, as Rotting Oranges is a Medium question.
   * Observe the image loaded in the question display area.

**Example of question with image (Updated)**
![withImage](https://user-images.githubusercontent.com/12153882/198883541-be4284c3-a660-4b36-8217-dac0cdf41e11.png)


**Example of question without image**
![withoutImage](https://user-images.githubusercontent.com/12153882/198320845-37df68ee-6845-45c2-8567-fd6b26d9c1bb.png)
